league_rpc_linux/__main__.py

```diff
@@ -116,18 +128,15 @@ def main(cli_args: argparse.Namespace):
                             time.sleep(10)
 
                 case "InLobby":
```

Comment:
Since there is now no code in here. how will we know what is happening when this case statement is executed.

Right now, it's just a sleep. Do we want to keep that in?
Should lcu driver perhaps only be executed when we reach this case statement to avoid that process to be running in the background while players are in game?

Reply:
I think we can just keep it running in the background as the WS will not get any updates (at least not for our purposes) and thus will not update RPC. 
There is no mesurable perfomance decrease. 
The Process Idles unless there is a data update (to the data which we subscribed to) from the WS. - No CPU time is used at all.
Everything else we dont subscribe to is not even sent by the API, it never gets passed through the ws, and doesnt reach the process - which definitely is super nice.